infra/scripts/sample-helper.profiles

```diff
@@ -0,0 +1,70 @@
+#!/bin/bash
+
+# example entry, actived by typing "example"
```

Comment:
This comment is confusing because it does not match the example entry, which would be activated by running `my-vch`. Left me a little confused as to how to select a profile.